KGTorrent/mk_preprocessor.py

```diff
@@ -0,0 +1,259 @@
+"""
+This module defines the class that handles the whole preprocessing of the MetaKaggle tables in order to ensure their
+entity integrity, referential integrity and domain integrity.
```

Comment:
What's the difference between _entity integrity_ and _domain integrity_?

Reply:
Entity integrity ensure that the primary key identify one specific tuple; we check that only the table 'ForumMessageVotes' need a drop of 'Id' duplicates.
The domain integrity ensure that data in a column are all of one specific type and, sometimes, in a specific range. The parsing of date columns and other fix of float columns fall into this case.

Reply:
Ok, that's fine.